backend/api/src/main/java/com/virtuslab/gitmachete/backend/api/IGitMacheteRepositorySnapshot.java

```diff
@@ -29,7 +30,14 @@ public interface IGitMacheteRepositorySnapshot {
   Option<IExecutionResult> executeMachetePreRebaseHookIfPresent(IGitRebaseParameters gitRebaseParameters)
       throws GitMacheteException;
 
-  OngoingRepositoryOperation getOngoingRepositoryOperation();
+  @Data
+  // So that Interning Checker doesn't complain about enum comparison (by `equals` and not by `==`) in Lombok-generated `equals`
+  @SuppressWarnings("interning:not.interned")
```

Comment:
```suggestion
  @SuppressWarnings("interning:unnecessary.equals")
```
(or both `not.interned` and `unnecessary.equals`? sth like that)